### DISCUSSION TITLE
Query about pretrained models

### DISCUSSION THREAD

**Discussion by sovit-123** [open-mmlab/mmdetection#7108] [General]
This is a great repo and helps a lot in many CV projects.
I have a simple question to ask. Are all the models in this project custom trained by the developers or are collected from different official sources like PyTorch models, etc?

Thanks.

**Comment by baibaidj** [comment]
I believe that the models in this repo are implemented and trained by the developers, including most of the backbones. You could refer to the README.md of each model for training log under the config folder. For instance, [https://github.com/open-mmlab/mmdetection/tree/master/configs/retinanet](url)

But there are also models and their weights imported (converted if necessary to be compatible to this code style in this repo) from other sources, such as [timm](https://github.com/open-mmlab/mmclassification/blob/e0edffb353e5ac3176140429a32688c64e55ac88/mmcls/models/backbones/timm_backbone.py) and [repvgg.](https://github.com/open-mmlab/mmclassification/tree/master/configs/repvgg). 
Hope it helps.
